components/org.wso2.carbon.identity.auth.otp.core/src/main/java/org/wso2/carbon/identity/auth/otp/core/constant/AuthenticatorConstants.java

```diff
@@ -0,0 +1,157 @@
+    }
+
+    /**
+     * Authentication flow scenarios associated with the authenticator.
+     */
+    public enum AuthenticationScenarios {
+
+        LOGOUT,
+        INITIAL_OTP,
+        RESEND_OTP,
+        SUBMIT_OTP,
+    }
+
+    /**
+     * Enum which contains the error codes and corresponding error messages.
+     */
+    public enum ErrorMessages {
+
+        ERROR_CODE_ERROR_GETTING_CONFIG("65001", "Error occurred while getting the authenticator " +
+                "configuration"),
+        ERROR_CODE_USER_ACCOUNT_LOCKED("65002", "Account is locked for the user: %s"),
+        ERROR_CODE_EMPTY_OTP_CODE("65003", "OTP token is empty for user: %s"),
+        ERROR_CODE_RETRYING_OTP_RESEND("65004", "User: %s is retrying to resend the OTP"),
+        ERROR_CODE_EMPTY_GENERATED_TIME("65005", "Token generated time not specified"),
+        ERROR_CODE_EMPTY_OTP_CODE_IN_CONTEXT("65006", "OTP token is empty in context for user: %s"),
+        ERROR_CODE_ERROR_GETTING_BACKUP_CODES("65007",
+                "Error occurred while getting backup codes for user: %s"),
+        ERROR_CODE_ERROR_GETTING_ACCOUNT_UNLOCK_TIME("65008",
+                "Error occurred while getting account unlock time for user: %s"),
+        ERROR_CODE_ERROR_UPDATING_BACKUP_CODES("65009",
+                "Error occurred while updating unused backup codes for user: %s"),
+        ERROR_CODE_ERROR_GETTING_MOBILE_NUMBER("65010",
+                "Error occurred while getting the mobile number for user: %s"),
+        ERROR_CODE_ERROR_GETTING_USER_REALM("65011",
+                "Error occurred while getting the user realm for tenant: %s"),
+        ERROR_CODE_ERROR_REDIRECTING_TO_LOGIN_PAGE("65012",
+                "Error occurred while redirecting to the login page"),
+        ERROR_CODE_ERROR_TRIGGERING_EVENT("65013",
+                "Error occurred while triggering event: %s for the user: %s"),
+        ERROR_CODE_ERROR_REDIRECTING_TO_ERROR_PAGE("65014",
+                "Error occurred while redirecting to the error page"),
+        ERROR_CODE_NO_USER_FOUND("65015", "No user found from the authentication steps"),
+        ERROR_CODE_EMPTY_USERNAME("65016", "Username can not be empty"),
+        ERROR_CODE_ERROR_GETTING_USER_STORE_MANAGER("65017",
+                "Error occurred while getting the user store manager for the user: %s"),
+        ERROR_CODE_GETTING_ACCOUNT_STATE("65018", "Error occurred while checking the account locked " +
+                "state for the user: %s"),
+        ERROR_CODE_OTP_EXPIRED("65019", "OTP expired for user: %s"),
+        ERROR_CODE_OTP_INVALID("65020", "Invalid code provided by user: %s"),
+        ERROR_CODE_ERROR_GETTING_FEDERATED_AUTHENTICATOR("65021", "Error occurred while getting IDP: " +
+                "%s from tenant: %s"),
+        ERROR_CODE_INVALID_FEDERATED_AUTHENTICATOR("65022", "No IDP found with the name : " +
```

Comment:
ERROR_CODE_INVALID_FEDERATED_AUTHENTICATOR("65022", "No IDP found with the name: " +

Reply:
Fixed with 6239874dcb1bf208bfccc91c0af83bf6b6747dbc